services/dy-jupyter-extensions/neuron/requirements/base.txt

```diff
@@ -1,88 +1,111 @@
 aiofiles==0.4.0
```

Comment:
I am a bit confused, I cannot find the source file, i.e. ``base.in`` in your repo... Did you by any chance forgot to version it?

Reply:
There is no base.in. I simply take what comes with the base image. ```make base```) and on top of that I add more requirements (prod only)

Reply:
After discussion, the requirement files shall follow rationale explained [in docs](https://github.com/ITISFoundation/osparc-simcore/blob/master/docs/python-dependencies.md) : base.txt shall be renamed and included in _base.in as required